test_istgt.sh

```diff
@@ -570,14 +570,28 @@ check_healthy_replica_count()
 {
```

Comment:
Please update the description of the function and also the function name.
 According to the new changes, it should be like `wait_for_healthy_replicas`.